Bottom: 7d4587bccea1c043bfcddcf0049807ca43edb3f4
Top:    e3f99d9e2a4ed93d8cdf0045c871754c0139d950
Author: Ian Kent <ikent@redhat.com>
Date:   2013-07-13 11:29:26 +0800

Refresh of autofs-5.0.7-fix-a-couple-of-compiler-warnings.patch

---

diff --git a/CHANGELOG b/CHANGELOG
index 37eac72..7749f01 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -54,6 +54,7 @@
 - misc man page fixes.
 - fix add null check in parse_server_string().
 - don't override LDFLAGS in make rules.
+- fix a couple of compiler warnings.
 
 25/07/2012 autofs-5.0.7
 =======================
